pkgs/development/tools/misc/gdb/default.nix

```
@@ -8,6 +8,7 @@
 
 , pythonSupport ? stdenv.hostPlatform == stdenv.buildPlatform && !stdenv.hostPlatform.isCygwin, python3 ? null
 , guile ? null
+, safePaths ? [ stdenv.cc.cc.lib ]
```

I would still prepend gdb's own default value. At least this is how I understood it from reading the autotools configuration.

```suggestion
, safePaths ? [ "$debugdir" "$datadir/auto-load" stdenv.cc.cc.lib ]
```

---

Indeed: `configure flags: [...] --with-auto-load-safe-path=\$debugdir:\$datadir/auto-load:/nix/store/<hash>-gcc-8.3.0-lib`.